aocd/runner.py

```diff
@@ -182,24 +194,42 @@ def main():
     sys.exit(rc)
 
 
-def _timeout_wrapper(f, capture=False, timeout=DEFAULT_TIMEOUT, **kwargs):
+def _timeout_wrapper(
+    f: Callable["_PS", _R],
+    capture: bool = False,
+    timeout: float = DEFAULT_TIMEOUT,
+    *args: "_PS.args",
```

Comment:
This is an extension of the method here that isn't reflected elsewhere. Either don't add `*args` or add `*args` to `run_with_timeout()` too. 

I'd lean towards *not* adding this, and focusing on _just_ type annotations for the existing functionality.